lesson10/main.js

```diff
@@ -62,7 +62,10 @@ window.onload = () => {
     }
   };
 
-  callLists().then((val) => {
+  const result = async () => {
```

Comment:
Good. I approve.

nits:
I would prefer a different name than "result".
For example, "init".

Reply:
@matsuwork
I fixed name, thank you for teaching me and your time :)
[75b7525](https://github.com/yuka830/js-lessons/pull/12/commits/75b75257f890f05989a4dc1c10c58f3745e6de70)
[CodeSandbox](https://codesandbox.io/s/lesson10-gn6i7?file=/main.js)
Last commit [4664130](https://github.com/yuka830/js-lessons/pull/12/commits/46641304373e21b99f09adef43bad8a946620735)